### PR TITLE
Fix crash at loading app analytics tabs

### PR DESCRIPTION
**Pull request by joshuali925**
### Description
For future refactoring: it would be better to use core chrome service to get current breadcrumbs, then append new ones to it, rather than passing down parentBreadcrumbs as props.

Passing parent down can be inconsistent (this case breadcrumb vs array<breadcrumbs>) and redundant

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/components/trace_analytics/components/dashboard/__tests__/__snapshots__/dashboard.test.tsx.snap [hunk 1]
```diff
@@ -305,7 +327,29 @@ exports[`Dashboard component renders dashboard 1`] = `
...
         "setBadge": [MockFunction],
         "setBrand": [MockFunction],
-        "setBreadcrumbs": [MockFunction],
+        "setBreadcrumbs": [MockFunction] {
+          "calls": Array [
+            Array [
+              Array [
+                undefined,
+                Object {
+                  "href": "#/trace_analytics/home",
```
Comment (rupal-bq) [inline public/components/trace_analytics/components/dashboard/__tests__/__snapshots__/dashboard.test.tsx.snap:336]: is `trace_analytics` in url works? I thought it was removed in earlier pr

**Review by anirudha — APPROVED** [review]
thanks

**Review by rupal-bq — APPROVED** [review]
LGTM

File: public/components/trace_analytics/components/dashboard/__tests__/__snapshots__/dashboard.test.tsx.snap (hunk 1 above)
Comment (joshuali925) [reply public/components/trace_analytics/components/dashboard/__tests__/__snapshots__/dashboard.test.tsx.snap:336]: that is probably from test files https://github.com/opensearch-project/dashboards-observability/blob/544c2f50521501c87deacf3136b0596686467821/public/components/trace_analytics/components/dashboard/__tests__/dashboard.test.tsx#L39

The unit test doesn't care what it should be in reality, as long as dummy arguments are all valid it serves its purpose

**Pull request merged**